style.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #170606;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+header {
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  background-image: url(./images/Illustration-header.png);
+  background-repeat: no-repeat;
+  background-size: 200px 300px;
+  background-position: 240px 390px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 30px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: white;
+  background: #3c3a39;
+}
+
+nav i:hover {
+  color: #bbbab8;
+  transition: 0.5s ease-in-out;
+  cursor: pointer;
+}
+
+nav h5:hover {
+  color: #bbbab8;
+  transition: 0.5s ease-in-out;
+  cursor: pointer;
+}
+
+section {
+  padding: 40px;
+  margin-top: 50px;
+}
```

Comment:
- [ ] [REQUIRED] Please remove your `padding` because header elements and social network icons should be located on the left side!

- ### Your Design ❌
![image](https://user-images.githubusercontent.com/63915024/185956913-1b8366ce-623c-44a5-872d-490dc1e167bf.png)

- ### Figma Design ✔️ 

![image](https://user-images.githubusercontent.com/63915024/185960636-a039c64d-64a1-42b9-a7f1-6a006a25542a.png)